utils/appliance.py

```diff
@@ -39,6 +39,25 @@
     from utils.hosts import setup_providers_hosts_credentials
 
 
+class logger_wrap(object):
```

Comment:
So, this could live in `utils.log` and the kwarg could be modifiable but log_callback by default...